lib/pager.gi

```diff
@@ -85,7 +85,11 @@ leave the 'Pager' and 'PagerOptions' preferences empty."
         return [ pager, options ];
       fi;
     fi;
-    return [ "builtin", [] ];
+    if IsBound(HPCGAP) then
+      return [ "less" , ["-f","-r","-a","-i","-M","-j2"] ];
```

Comment:
Perhaps a comment here that explains what's going on? Can be a short single line comment